Source/Extensions/Blazorise.Components/DropdownList.razor.cs

```diff
@@ -88,6 +88,11 @@ protected bool IsSelected( TValue value )
     /// </summary>
     [Parameter] public Color Color { get; set; }
 
+    /// <summary>
+    /// Defines the size of toggle button.
+    /// </summary>
+    [Parameter] public Size Size { get; set; }
```

Comment:
Would it be better to name it `DropdownToggleSize` or something along those lines? 

One could assume this to be the `Size` or the List or the item incorrectly?

Other than that I guess it's fine to make this available if it's a necessity. 

@stsrki Ideally in the future we can figure a way to pass a bag of parameters to inner components. (I think we have an issue open somewhere for that.)

Reply:
I agree, that DropdownToggleSize seems more appropriate in this context.

> @stsrki Ideally in the future we can figure a way to pass a bag of parameters to inner components. (I think we have an issue open somewhere for that.)

We do have an issue. The main problem is that whatever route we go it feels like the inner API of the internal component structure will leak out. And in some cases, it is unique to certain providers.